### ISSUE TITLE
[TODO] Complete the Groups component

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#76]
**Is your feature request related to a problem?**
 No

Complete the Groups component using Material UI v5.
It should open via clicking on the Groups button im the left SideNav/Sidebar.

Route should be /groups/ 

Make some facilities for CRUD operations for Groups as well as adding Admin and Moderators and ìnviting users

**Describe the solution you'd like**
As above

**Describe alternatives you've considered**
As above

**Additional context**
Will add some screenshots to be used as inspiration for this component tomorrow when able to return to the PC.

**Comment by gbowne1** [comment]
The route /groups/create/ should show something like this:

![Screenshot 2023-02-22 at 11-41-14 (3) Create group Facebook](https://user-images.githubusercontent.com/47549872/220741166-b817ea5b-4e85-48df-a5fd-70afe24cc1a9.png

**Comment by gbowne1** [comment]
Typical Creating Group Workflow:
- Name Group
- Map or Pin Group Location
- Set group privacy
- Invite members
- Describe Group (about)
- Set up preview with Banner, etc.
- Initial post
- Tag group

**Comment by gbowne1** [comment]
Context menu ... for Groups
![Screenshot 2023-02-28 at 18-09-47 Join Nextdoor an app for neighborhoods where you can get local tips buy and sell items and more](https://user-images.githubusercontent.com/47549872/222026651-37baffc5-05a9-4323-b3a6-e2392ebd4a99.png)

**Comment by gbowne1** [comment]
I want to target with this:

* YouTubers
* Tech including Programmers
* Seniors
* Community/Neighborhood
* Middle age Singles
* Mentorships

**Comment by deecodess** [comment]
Hey! I would love to work on this.

**Comment by gbowne1** [comment]
Ok, @deecodess.  I can assign you.

**Comment by gbowne1** [comment]
@deecodess are you still working on this?

**Comment by deecodess** [comment]
@gbowne1 Yes! My exams just got finished. I am still working on this. Thank you for your patience.

**Comment by gbowne1** [comment]
Ok, just checking in.

**Comment by gbowne1** [comment]
@deecodess it's been another month. Are you still working on this? I can reassign if not.

**Comment by gbowne1** [comment]
@manuel12 @pawel975 @balajik
